python/src/trezorlib/cli/settings.py

```diff
@@ -132,6 +132,20 @@ def homescreen(client, filename):
     return device.apply_settings(client, homescreen=img)
 
 
+@cli.command()
+@click.argument("allow", type=click.Choice(("on", "off")))
+@with_client
+def unsafe_prompts(client, allow):
+    """Allow or disallow unsafe prompts.
+
+    This is a power-user feature. With unsafe prompts enabled, Trezor will ask the user
+    to confirm possibly dangerous actions instead of rejecting them outright.
+    Use with caution.
+    """
```

Comment:
I suggest we display this description to the user in CLI. Also with a `Continue? [y/N]` to make this a bit more scarier :).

Reply:
I considered that, but OTOH the scary warning is already displayed on the Trezor screen?

Reply:
Still - we can be more descriptive here.